jetty-osgi/test-jetty-osgi-server/pom.xml

```diff
@@ -66,9 +66,6 @@
             compilation time. -->
             <_nouses>true</_nouses>
             <Import-Package>
-              ${osgi.slf4j.import.packages},
```

Comment:
Did you mean to remove the slf4j imports here?

Reply:
Yes. Putting in the `osgi.slf4j.import.packages` statement is _only_ necessary if the code in that module does any logging. This module does not.